ui/components/ui/form-field/form-field.js

```diff
@@ -67,13 +73,37 @@ export default function FormField({
             {tooltipText && (
               <InfoTooltip position="top" contentText={tooltipText} />
             )}
+            {tooltipContentText && (
+              <Box display={DISPLAY.INLINE_BLOCK}>
+                <Tooltip
+                  interactive
+                  position="top"
+                  html={
+                    <Typography
+                      variant={TYPOGRAPHY.H7}
+                      margin={3}
+                      color={COLORS.TEXT_ALTERNATIVE}
+                      className="form-field__heading-title__tooltip"
+                    >
+                      {tooltipContentText}
+                    </Typography>
+                  }
+                >
+                  {tooltipIcon ? (
+                    <i className="fa fa-exclamation-triangle form-field__heading-title__tooltip__warning-icon" />
+                  ) : (
+                    <i className="fa fa-question-circle" />
+                  )}
+                </Tooltip>
+              </Box>
+            )}
           </div>
           {titleDetail && (
             <Box
               className="form-field__heading-detail"
               textAlign={TEXT_ALIGN.END}
-              marginBottom={3}
-              marginRight={2}
+              marginBottom={customSpendingCapText || tooltipContentText ? 2 : 3}
+              marginRight={customSpendingCapText || tooltipContentText ? 0 : 2}
```

Comment:
Another pattern we could add to this component is prop objects for each of the individual components so they are easy to change in their use case

```suggestion
              marginBottom={3}
              marginRight={2}
              {...titleDetailProps}
```

Reply:
@georgewrmarshall Done [here](https://github.com/MetaMask/metamask-extension/pull/15522/commits/190f93d7a3b816927c6d40f7039080fdc87be55b).